plot-gui-lib/plot/private/gui/plot3d.rkt

```diff
@@ -189,8 +196,8 @@
                 #:z-min [z-min #f] #:z-max [z-max #f]
                 #:width [width (plot-width)]
                 #:height [height (plot-height)]
-                #:angle [angle (plot3d-angle)]
-                #:altitude [altitude (plot3d-altitude)]
+                #:angle [angle #f]
+                #:altitude [altitude #f]
```

Comment:
Note: if these are not `#false`, then the `#:az` and `#:alt` arguments are unused --- even though L246 and others try to use them.